### ISSUE TITLE
Create Color from (hex) Integers

### ISSUE THREAD

**Issue by rylev** [nannou-org/nannou#323]
It would be nice to be able to convert hex integers into colors so the one could do the following:

```rust
draw.ellipse()
            .x_y(x, y)
            .radius(0.5)
            .color(0x2ECC40)
```

**Comment by MacTuitui** [comment]
I am also for it, but in the mean time, this is how I do the manual conversion if it helps (disclaimer: I'm still a beginner so there might be better ways):

```rust
let colors: [u32; 13] = [0xFF70382E,0xFF48221F,0xFF25242C,0xFF5B413F,0xFF2C474C,0xFF306C6F,0xFF555656,0xFF288C8E,0xFF0AF7F7,0xFF1FAEAF,0xFF19D3D3,0xFF587473,0xFFB2DBDD];
let colorsv = colors.to_vec();
let colsrgb : Vec<Rgb> = colorsv.into_iter().map(|c| {  
    let blue: u8 = (c & 0xFF) as u8;    
    let green: u8 = (( c >> 8 ) & 0xFF) as u8;
    let red: u8 = (( c >> 16 ) & 0xFF) as u8;
    let c = Srgb::new_u8(red, green,blue);
    c.into()
}).collect();
```

Then I have an array of colors that nannou accepts (or that you can `into()` if needed).

**Comment by cronokirby** [comment]
Doesn't the following function kind of satisfy this?
https://docs.rs/nannou/0.11.1/nannou/ui/prelude/color/fn.rgb_bytes.html

then you could do `.color(rgb_bytes(0x2E, 0xCC, 0x40))`
you could also add similar functions that take a single u32, e.g. `rgb_u32`

**Comment by jli** [comment]
I think this has been fixed by #545?

**Comment by mitchmindtree** [comment]
Closed via #545
